docs/contributing/forem.md

```diff
@@ -0,0 +1,180 @@
+and
+[type: bug](https://github.com/thepracticaldev/dev.to/issues?q=is%3Aissue+is%3Aopen+label%3A%22type%3A+bug%22)
+are up for grabs.
+
+- [good first issue](https://github.com/thepracticaldev/dev.to/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22+)
+  issues are meant for newer developers.
+- [difficulty: easy](https://github.com/thepracticaldev/dev.to/issues?q=is%3Aopen+is%3Aissue+label%3A%22difficulty%3A+easy%22)
+  issues are usually confined to isolated areas of existing code.
+- [difficulty: medium](https://github.com/thepracticaldev/dev.to/issues?q=is%3Aopen+is%3Aissue+label%3A%22difficulty%3A+medium%22)
+  issues can sometimes entail new features and might affect a significant area
+  of the codebase, but aren't overly complex.
+- [difficulty: hard](https://github.com/thepracticaldev/dev.to/issues?q=is%3Aopen+is%3Aissue+label%3A%22difficulty%3A+hard%22)
+  issues will typically be far-reaching, and might need architecture decisions
+  during implementation. This label might also denote highly complex issues.
+
+PRs without an associated issue may still be merged, but the core team will
+focus on changes that solve existing issues. We strongly encourage creating an
+issue before working on a PR!
+
+When in doubt, ask a
+[core team member](https://github.com/thepracticaldev/dev.to/#core-team) by
+mentioning us on the issue.
+
+**Documentation** is almost always a great place to start contributing to a new
+project. DEV is an Open Source, community-driven project. Therefore, providing
+and maintaining quality documentation is one of our most important jobs. You can
+find more information in our
+[docs guide](https://docs.dev.to/contributing/docs)!
+
+**Refactoring** (improving the code without modifying behavior) is a great place
+to help out! Generally speaking, you can rely on existing tests to ensure that
+your refactor doesn't introduce any unexpected behavior. If an area isn't well
+tested, you might be asked to include a regression test with your refactoring
+PR. Refactors can touch many files, so we encourage breaking big changes into
+small PRs.
+
+**Fixing bugs** is a super fast way to improve the experience for our users!
+When you're fixing bugs, we appreciate communication in a GitHub issue. If an
+issue exists, please claim that issue and link it in your PR, otherwise creating
+an issue is the best first step! Be sure to surround bug fixes with ample tests;
+bugs are magnets for other bugs. Write tests around bugs!
+
+**Building features** requires a lot of communication, but we'd love to have
+your help with this too! Features tend to be subjective and might spur some
+debate. Be sure to create an issue for new features before getting started! If
+your feature involves design changes, including design mockups can be very
+helpful. As always, when in doubt, ask!
+
+### How to contribute
+
+1. [Fork the project](https://docs.dev.to/getting-started/forking/) and clone it
```

Comment:
Should these links just be internal ones rather than the full `docs.dev.to` link? Not a blocker just curious why the full path :)

Reply:
They probably should be internal, but I think links need to be redone for most of the docs site, so I didn't change these when I moved them from `CONTRIBUTING.md` :sweat_smile: